.circleci/config.yml

```diff
@@ -20,6 +20,9 @@ jobs:
     - run:
         name: PyTest with linters
         command: PATH="venv/bin:$PATH" script/entrypoint
+    - run:
+        name: Verify that all generated files are committed
+        command: git diff --exit-code target/
```

Comment:
I'd prefer that we not check in the generated files in order to keep the repository cleaner. In order to make that happen, we'd need to:

- Modify the .gitignore to ignore the `target/` directory
- Modify the CircleCI flow to make the built sql files available to the `deploy` step

For the CircleCI piece, I think we should add a [persist_to_workspace](https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace) step to the `build` job that persists the built contents of `target`, and then [attach_workspace](https://circleci.com/docs/2.0/configuration-reference/#attach_workspace) for the `deploy` step to make those files available when building the docker container.